lib/kaguya/core.ex

```diff
@@ -15,6 +15,7 @@ defmodule Kaguya.Core do
   defp password, do: Application.get_env(:kaguya, :password)
   defp use_ssl, do: Application.get_env(:kaguya, :use_ssl)
   defp reconnect_interval, do: Application.get_env(:kaguya, :reconnect_interval)
+  defp server_timeout, do: Application.get_env(:kaguya, :server_timeout)
```

Comment:
Since you're relying on a new config variable, could you add it to the default config or give it a default value here?